dependencies/bz2/meson.build

```diff
@@ -0,0 +1,14 @@
+# bz2 Dependency File
```

Comment:
This is not a dependency upstream anymore (removed by https://github.com/NixOS/nix/commit/15f4d4fd437070e898982c510228b6e8eea17b7d).